Client/Frontend/TabBar/SearchViewController.swift

```diff
@@ -151,7 +151,10 @@ extension SearchViewController: UITableViewDelegate {
             let suggestion = searchSuggestions[indexPath.row]
 
             // Assume that only the default search engine can provide search suggestions.
```

Comment:
Please move this comment into the if block with the other line, since it applies to that line. Also, please remove braces around if condition. So:
```
if url == nil {
    // Assume that only the default search engine can provide search suggestions.
    url = searchEngines?.defaultEngine.searchURLForQuery(suggestion)
}
```